### DISCUSSION TITLE
rendering error

### DISCUSSION THREAD

**Discussion by FizzYam** [vercel/community#3835] [Help]
### Summary

Strange characters are displayed when rendering

The same thing happens in other projects as well

It appears only in the production environment, but not in the development environment.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/34015900/8134d363-7c0e-45fa-935d-1744dd762e8e)

**Comment by FizzYam** [comment]
There is a typo so I am correcting it.

It appears only in the production environment, but not in the development environment.

=>

It only appears in the development environment and does not appear in the operating environment.

**Comment by amyegan** [comment]
Hey @FizzYam. Do you have a public repo or [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to help with debugging?
